spec/services/idv/profile_step_spec.rb

```diff
@@ -134,16 +119,11 @@ def build_step(params)
       expect(idv_session.profile_confirmation).to be_nil
     end
 
-    it 'increments attempts count if the form is valid' do
+    it 'increments attempts count' do
       step = build_step(user_attrs)
       expect { step.submit }.to change(user, :idv_attempts).by(1)
     end
 
-    it 'does not increment the attempts count if the form is not valid' do
```

Comment:
Now that we've separated the form submission from the vendor validation, should we make sure we have a controller or integration test that makes sure the attempts count is only incremented if the form submission passed?